services/web/client/source/class/osparc/desktop/StudyEditor.js

```diff
@@ -232,23 +239,32 @@ qx.Class.define("osparc.desktop.StudyEditor", {
         })
         .finally(() => this._hideLoadingPage());
 
-      this.__workbenchView.setStudy(study);
-      this.__slideshowView.setStudy(study);
+      this.__updatingStudy = 0;
     },
 
     __reloadSnapshotsAndIterations: function() {
-      const store = osparc.store.Store.getInstance();
-      store.invalidate("snapshots");
-      store.invalidate("iterations");
-
-      const study = this.getStudy();
-      study.getSnapshots()
-        .then(snapshots => {
-          store.setSnapshots(snapshots);
-          if (snapshots.length) {
-            study.getIterations()
-              .then(iterations => {
-                store.setIterations(iterations);
+      osparc.utils.DisabledPlugins.isVersionControlDisabled()
+        .then(isVCDisabled => {
```

Comment:
does the UI hides/disables the "snapshot" and "iterators" sections as well

Reply:
Yes, that goes in ```services/web/client/source/class/osparc/desktop/WorkbenchView.js```